frontend/src/components/AcitveMembership/ActiveMembership.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+// @material-ui/core components
+import { Grid } from "@material-ui/core";
+// core components
```

Comment:
Doesn't add any value, try to avoid comments and replace with code that "says what it does".